### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by Avivsalem**

### CODE REVIEW

File: docs/contribute/code.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing Code
+
```
Comment (noamisraeli) [inline docs/contribute/code.md:2]: Do we want to add an issue for each feature and like it to the pr?

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+[![last commit](https://badgen.net/github/last-commit/Avivsalem/MessageFlux/main)](https://github.com/Avivsalem/MessageFlux/commit/main)
+[![tests](https://github.com/AvivSalem/MessageFlux/actions/workflows/tests.yml/badge.svg)](https://github.com/AvivSalem/MessageFlux/actions/workflows/tests.yml?query=branch%3Amain)
+[![Documentation Status](https://readthedocs.org/projects/messageflux/badge/?version=latest)](https://messageflux.readthedocs.io/en/latest/?badge=latest)
+[![pypi version](https://badgen.net/pypi/v/MessageFlux)](https://pypi.org/project/messageflux/)
+[![python compatibility](https://badgen.net/pypi/python/MessageFlux)](https://pypi.org/project/messageflux/)
+[![downloads](https://img.shields.io/pypi/dm/messageflux)](https://pypi.org/project/messageflux/)
+
+messageflux is a framework for creating long-running services, that read messages from devices, and handles them.
+
+The devices in this package are meant to be composable - meaning that added functionality can come from wrapping devices
```
Comment (noamisraeli) [inline docs/index.md:14]:
```suggestion
Devices are compostable components - meaning that added functionality can come from wrapping devices
```

File: docs/contribute/code.md (hunk 1 above)
Comment (Avivsalem) [reply docs/contribute/code.md:2]: i don't know. should we? doesn't seem too important

File: docs/tutorial/services.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# MessageFlux Services
```
Comment (noamisraeli) [inline docs/tutorial/services.md:1]:
```suggestion
# Services
```

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+### Extra Requirements (Example)
+
+```console
+$ pip install messageflux[rabbitmq]
+```
+
+!!! tip "Tip"
+This is a tip
```
Comment (noamisraeli) [inline docs/index.md:34]:
```suggestion
    This is a tip
```

Make sure locally that this fixes the problem

File: docs/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+mkdocs-material
+mkdocstrings[python]
+markdown-include
+mkdocs-gen-files
+mkdocs-literate-nav
+mkdocs-section-index
+Black
```
Comment (noamisraeli) [inline docs/requirements.txt:7]: What is this ?

Comment (Avivsalem) [reply docs/requirements.txt:7]: mkdocs extensions needed for the reference generation i wrote. (by the recipe)

File: .github/workflows/deploy.yml [hunk 6]
```diff
@@ -64,3 +64,10 @@ jobs:
         run: |
           twine upload -u __token__ -p ${{ secrets.MESSAGEFLUX_PYPI_TOKEN }} dist/*
 
+      - name: Create Release
```
Comment (noamisraeli) [inline .github/workflows/deploy.yml:67]: What is this ?

**Review by noamisraeli — APPROVED** [review]

**Pull request merged**